test/Feature/InsertSpec.hs

```diff
@@ -441,9 +437,6 @@ spec actualPgVersion = do
 
   describe "Row level permission" $
     it "set user_id when inserting rows" $ do
-      post "/postgrest/users" [json| { "id":"jdoe", "pass": "1234", "role": "postgrest_test_author" } |]
```

Comment:
After changing the ghc flags to `-Werror` for the test suite as well, I got some errors here. I removed those two lines.... and the test still passes. That kind of worries me about the whole test...

I will have to have a closer look at the whole thing.

Reply:
I don't think those two lines ever served any purpose. The route `/postgrest/users` certainly does not exist, so this request would always fail. But since the result of the request was never used or checked for, this wasn't a problem - until the compiler complained about the unused result. Should be safe to remove.

Reply:
@wolfgangwalther :O These are leftovers of a really old auth implementation. Nice that the coverage caught this. Tests should have failed.